anemometer/app/price-feed.js

```diff
@@ -0,0 +1,59 @@
+const ethers  = require('ethers');
+const fs      = require('fs');
+const axios = require('axios');
+
+ 
+const root                = 'iexec_out';
+const determinismFilePath = `${root}/determinism.iexec`;
+const callbackFilePath    = `${root}/callback.iexec`;
+const errorFilePath       = `${root}/error.iexec`;
+
+/*****************************************************************************
+ *                                  CONFIG                                   *
+ *****************************************************************************/
+const APIKEY = '87a1fd74e099f170b76906e87e19e9d9';
+
+/*****************************************************************************
+ *                                 ARGUMENTS                                 *
+ *****************************************************************************/
+
+var [ lati , longi] = process.argv.slice(2).map(s => s);
+console.log(longi);
+console.log(lati);
+/*****************************************************************************
+ *                                HTTP QUERY                                 *
+ *****************************************************************************/
+
+let path = `https://api.openweathermap.org/data/2.5/weather?lat=${lati}&lon=${longi}&APPID=${APIKEY}`;
```

Comment:
This is good but not deterministic. I would suggest replacing lat/lon with lat/lon/time and use the endpoint:
http://history.openweathermap.org/data/2.5/history/city?lat={lat}&lon={lon}&type=hour&start={start}&cnt={cnt}

Reply:
I have switched to https://api.darksky.net/forecast/${DARKSKYAPIKEY}/${lati},${longi},${timestamp}?exclude=hourly,daily,flags
which take in time and returns historical data, openweather historical data endpoint was a paid service.